tests/test_manure_module/test_manure_management_pen.py

```diff
@@ -0,0 +1,115 @@
+from typing import Callable
+from typing import Dict
+
+import pytest
+from pytest import approx
+from pytest_mock import MockerFixture
+
+from RUFAS.routines.animal.life_cycle.calf import Calf
+from RUFAS.routines.animal.life_cycle.cow import Cow
+from RUFAS.routines.animal.life_cycle.heiferI import HeiferI
+from RUFAS.routines.animal.life_cycle.heiferII import HeiferII
+from RUFAS.routines.animal.life_cycle.heiferIII import HeiferIII
+from RUFAS.routines.animal.pen import Pen
+from RUFAS.routines.manure.manure.pen_manure import PenManure
+from RUFAS.routines.manure.pen.manure_management_pen import ManureManagementPen
+
+
+def test_manure_management_pen_init(mocker: MockerFixture,
+                                    mock_pen: Pen,
+                                    mock_cow: Cow,
+                                    generate_animal_manure: Callable[[float], Dict[str, float]]) -> None:
+    """Unit test for function __init__ in file manure_management_pen.py"""
+
```

Comment:
Be more descriptive about what your tests are doing. 
```suggestion
    """Check that manure management is properly initialized, with correct values"""
```

I'd recommending all your test doscstrings to be more descriptive, but I'll only comment once.